src/pages/Profile.css

```diff
@@ -0,0 +1,66 @@
+.formContainer {
+    border: solid #c49e34;
+    border-radius: 5%;
+    width: 50%;
+    padding: 0.8rem;
+    margin: auto;
+    margin-top: 2%;
```

Comment:
you should use rem for margins, and no more than 4rem for margin: if you need more, you're facing a layout problem, not spacing. try flexbox/grid on the container instead